simple_settings/dynamic_settings/__init__.py

```diff
@@ -7,14 +7,26 @@
 )
 
 DYNAMIC_SETTINGS_MAPPING = {
-    'consul': 'simple_settings.dynamic_settings.consul_reader',
-    'database': 'simple_settings.dynamic_settings.database_reader',
-    'memcached': 'simple_settings.dynamic_settings.memcached_reader',
-    'redis': 'simple_settings.dynamic_settings.redis_reader',
-    's3': 'simple_settings.dynamic_settings.s3_reader'
+    'consul': 'simple_settings.dynamic_settings.consul_reader.Reader',
+    'database': 'simple_settings.dynamic_settings.database_reader.Reader',
+    'memcached': 'simple_settings.dynamic_settings.memcached_reader.Reader',
+    'redis': 'simple_settings.dynamic_settings.redis_reader.Reader',
+    's3': 'simple_settings.dynamic_settings.s3_reader.Reader'
 }
 
 
+class InvalidDynamicSettingsReaderPath(RuntimeError):
+
+    def __init__(self, path):
+        super(InvalidDynamicSettingsReaderPath, self).__init__(self)
+        self.path = path
+
+    def __str__(self):
+        return 'The path of dynamic settings reader [{}] are invalid'.format(
```

Comment:
💅 
I think the correct would be 
`The path of dynamic settings reader [{}] is invalid`

Reply:
you're right